packages/flutter_test/lib/src/window.dart

```diff
@@ -271,6 +271,15 @@ class TestWindow implements ui.SingletonFlutterWindow {
     platformDispatcher.onTextScaleFactorChanged = callback;
   }
 
+  @override
+  bool get brieflyShowPassword => _brieflyShowPasswordTestValue ?? platformDispatcher.brieflyShowPassword;
+  bool? _brieflyShowPasswordTestValue;
+  /// Hides the real [brieflyShowPassword] and reports the given
+  /// `brieflyShowPassword` instead.
```

Comment:
Nit: Maybe say `brieflyShowPasswordTestValue` instead of `brieflyShowPassword`?

Reply:
Ah right good catch!